src/wh_client_cryptocb.c

```diff
@@ -104,6 +104,8 @@ int wolfHSM_CryptoCb(int devId, wc_CryptoInfo* info, void* inCtx)
             dataSz = sizeof(packet->cipherAesCbcReq) +
                 info->cipher.aescbc.aes->keylen + AES_IV_SIZE +
                 info->cipher.aescbc.sz;
+            if (dataSz > WH_COMM_DATA_LEN)
+                return CRYPTOCB_UNAVAILABLE;
```

Comment:
Shouldn't this be a BAD_ARGS return?  It's likely the local SW crypto won't be able to handle the call.